src/iso19111/io.cpp

```diff
@@ -8701,27 +8718,38 @@ PROJStringParser::createFromPROJString(const std::string &projString) {
         const std::string &stepName = d->steps_[0].name;
         if (ci_starts_with(stepName, "epsg:") ||
             ci_starts_with(stepName, "IGNF:")) {
+
+            /* We create a new context so as to avoid messing up with the */
+            /* errorno of the main context, when trying to find the likely */
+            /* missing epsg file */
+            auto ctx = proj_context_create();
```

Comment:
Is there a way to pass the context passed in from `proj_create` here? That would definitely help from the perspective of the developer who passed in a context with specific settings configured & search paths set.

Reply:
Ah, I see. Nevermind. Looks like it copies those settings.